free-programming-books.md

```diff
@@ -846,8 +846,6 @@
 ### Basic
 
 * [10 PRINT CHR$(205.5+RND(1)); : GOTO 10](http://10print.org) - Nick Montfort, Patsy Baudoin, John Bell,Ian Bogost, Jeremy Douglass, Mark C. Marino, Michael Mateas, Casey Reas, Mark Sample, Noah Vawter
-* [A beginner's guide to Gambas](http://distro.ibiblio.org/vectorlinux/Uelsk8s/GAMBAS/gambas-beginner-guide.pdf) - John W. Rittinghouse (PDF)
```

Comment:
This link worked for me, maybe it was a temporary failure?

Reply:
The ibiblio link works, but http://distro.ibiblio.org/vectorlinux seems to have no affiliation with the author or subject of the book. I looked at the rest of the files in there, as well as the directory structure, and it all seemed unrelated. I also tried to find other copies of the book, since it’s supposed to be free, but found no other links. I found a listing on Google books but it only had the title and author and did not include the ebook. Clicking through to the ibiblio link also has that watermark running down the side.

So um I’m not sure if it’s meant to be still accessible or it’s a fluke cos someone uploaded a copy for themselves somewhere and it got picked up by ibiblio? Happy to add it back in if you think it’s fine :)

Reply:
ibiblio provides server space to a number of non-profit efforts such as project gutenberg, and various open source software projects. It's not on open file hosting service like those we discourage. So it should be left in.

Reply:
ah I see, added back in!